app/components/agenda/agenda-header/agenda-check.js

```diff
@@ -69,6 +81,28 @@ export default class AgendaHeaderAgendaCheck extends Component {
     return null;
   }
 
+  getNewPieces = task(async () => {
+    const agendaitems = await this.args.agenda.agendaitems;
+    const previousAgenda = await this.args.agenda.previousVersion;
+    const pieces = [];
+    const agendaitemNewPieces = agendaitems.map(async (agendaitem) => {
```

Comment:
Was concerned whether this would be too many network calls when loading a big agenda, but I tested it on acceptance with 100+ agendaitems, and all in all it was doable. Since we perform these same calls when loading the agenda overview, it seems they are still cached by the time we load the print view or agenda-check view. 👍